include/aws/io/tls_channel_handler.h

```diff
@@ -234,6 +243,36 @@ AWS_IO_API int aws_tls_ctx_options_init_client_mtls(
     const struct aws_byte_cursor *cert,
     const struct aws_byte_cursor *pkey);
 
+#    ifdef __APPLE__
```

Comment:
Rather than more constructors -- which in this case mirror existing ones but with an additional parameter -- maybe it would be better to just add a set method for the keychain path ala aws_tls_ctx_options_set_alpn_list?

Reply:
That sounds better than the extra constructors, I'll do that instead.